evaluate.py

```diff
@@ -735,7 +743,7 @@ def __init__(self, shape_dict, layout="NCHW", preproc=None):
 
     def Validate(self, m, ref_outputs=[]):
         if isinstance(m, tvm.runtime.vm.VirtualMachine) or isinstance(m, tvm.runtime.profiler_vm.VirtualMachineProfiler):
-            tvm_output = m.get_outputs()[0]
+            tvm_output = m.invoke("main", **data)
```

Comment:
I'm confused about that. Why we should infer the network one more time for getting outputs?

Reply:
I believe that is because we run "invoke_stateful" before, which "Invoke a function and _ignore the returned result._" (CC from tvm/python/tvm/runtime/vm.py#invoke_stateful). Besides that, `m.get_outputs(0)` should do the work, but  for some reason it is not returning correct outputs.